module-2/backend/week_four.md

```diff
@@ -10,14 +10,26 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 * What is a cookie?
+It's a little chunk of information that is set when a person's computer connects to a site. It is then passed back and forth with the as part of the HTTP request/response.
```

Comment:
a small piece of data sent from a website and stored on the user's computer by the user's web browser while the user is browsing.